api/swagger-spec/resourceListing.json

```diff
@@ -24,6 +24,14 @@
    {
     "path": "/apis/extensions",
     "description": "get information of a group"
+   },
+   {
+    "path": "/apis/autoscaling/v1",
```

Comment:
Fixed. I remember your comment. I ask there what should I do and you explained above:)